examples/Demo/FluentUI.Demo.Client/Layout/DemoAsidePanel.razor.cs

```diff
@@ -0,0 +1,38 @@
+// ------------------------------------------------------------------------
+// MIT License - Copyright (c) Microsoft Corporation. All rights reserved.
+// ------------------------------------------------------------------------
+
+using FluentUI.Demo.DocViewer.Extensions;
+using FluentUI.Demo.DocViewer.Models;
+using Microsoft.AspNetCore.Components;
+
+namespace FluentUI.Demo.Client.Layout;
+
+public partial class DemoAsidePanel
+{
+    /// <summary />
+    [Inject]
+    public NavigationManager Navigation { get; set; } = default!;
+
+    /// <summary />
+    [Inject]
+    public Demo.DocViewer.Services.DocViewerService DocViewerService { get; set; } = default!;
```

Comment:
Any particular reason for not creating an `@using`?

Reply:
Nop... Will fixed